### DISCUSSION TITLE
how to use two diffrent button control two diffrent bank

### DISCUSSION THREAD

**Discussion by fareign** [tttapa/Control-Surface#1071] [General]
for example:
I want use button 10 to control bank1,with ccnote 1 2 3 4 5 6 7 8 
and use button 11 to control bank2 with ccencoder 

how to do ? give me some mind.
thanks

**Comment by tttapa** [comment]
You can just instantiate multiple banks and multiple selectors in the same sketch. What have you tried, and why didn't that work?

**Reply by fareign** [reply]
> You can just instantiate multiple banks and multiple selectors in the same sketch. What have you tried, and why didn't that work?

i see thanks
